static/styles/lib/base.scss

```diff
@@ -160,6 +160,7 @@ footer {
 
 		@include media-breakpoint-down(xs) {
 			background: $colorBeige;
+			margin-top: 50px;
```

Comment:
this coudl be moved to a constant

Reply:
like $margin-top-50 ?

Reply:
more than related to the grid system margin-top: $2gridUnits

Reply:
Why? It's that specific here. 
In addition, there is already bootstrap stuff like `mt-1`, `mt-2`, ...

Reply:
I would not do that, constant for a specific margin-top value? nah...

Reply:
If we follow a global grid there are no specific margins. But its just a proposal, can easily changed later. and maybe something that we need to discuss with mona